run_autoformat.py

```diff
@@ -6,14 +6,51 @@
 it pushes a new commit to your pull request and aborts the current build.
 """
 
+import os
 import sys
 
 from travistooling import branch_name, get_changed_paths, git, make
 
 
+def _run_task_for_extension(extension, task):
```

Comment:
Really nice.

I don't like that this references travis env vars from a script in the root of the project but not blocking for merge.